src/comment/comment.controller.ts

```diff
@@ -27,8 +27,23 @@ export class CommentController {
   
 
   @Get()
-  findAll() {
-    return this.commentService.findAll();
+  @ApiOperation({ summary: 'Get all comments' })
```

Comment:
I think you designed this function to get all the comments of a post, right? If so then in ApiOperation you should write: Get all comments of a post.